src/components/labelledInputTextField/labelledInputTextField.jsx

```diff
@@ -0,0 +1,36 @@
+import "./labelledInputTextField.scss";
+
+const LabelledInputTextField = ({
+  uniqueInputFieldHtmlName,
```

Comment:
this prop represents an id, so perhaps just call it `id` or `uniqueId` for simplicity